### DISCUSSION TITLE
Only search starred repos

### DISCUSSION THREAD

**Discussion by abdurraheemali** [community/community#78306] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

In Advanced Code Search, I would like the ability to filter results to only starred repositories. This would integrate the "search stars" experience with the rest of github's search workflows.

![image](https://github.com/community/community/assets/98995519/6e2aef7a-b9e8-43a0-8af1-6df9b9e26bd5)

**Comment by look** [comment, marked as answer]
Thanks for the feature suggestion, that is an interesting idea.

**Reply by punkpeye** [reply]
was this ever implemented?
